test/test_torch.py

```diff
@@ -4224,11 +4224,13 @@ def test_serialization(self):
         b += [(t1.storage(), t1.storage(), t2.storage())]
         b += [a[0].storage()[0:2]]
         for use_name in (False, True):
-            with tempfile.NamedTemporaryFile() as f:
-                handle = f if not use_name else f.name
-                torch.save(b, handle)
-                f.seek(0)
-                c = torch.load(handle)
+            f = tempfile.NamedTemporaryFile(delete=False)
+            handle = f.name if use_name else f
+            torch.save(b, handle)
+            f.seek(0)
+            c = torch.load(handle)
+            f.close()
+            os.unlink(f.name)
```

Comment:
This should be at least wrapped in a try finally block. We don’t want to leave anything on disk if a test fails